### PR TITLE
docs: compress & improve user management docs

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

The user management docs are now one page, because the division between accounts and aliases is useless because there simply isn't enough content to justify the split. I improved and updated the text a bit.

Follow-Up to #3219

## Type of change

- [x] This change is a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by georglauterbach** [comment]
This PR can be merged at any time :)

File: docs/content/config/user-management.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# User Management
+
+## Accounts
+
+Users (email accounts) are managed in `/tmp/docker-mailserver/postfix-accounts.cf`. The best way to manage accounts is to use the reliable `setup` script inside the container. Just run `docker exec <CONTAINER NAME> setup help` and have a look at the section about subcommands, specifically the `email`-subcommand.
```
Comment (polarathene) [inline docs/content/config/user-management.md:5]:
```suggestion
Users (email accounts) are managed in `/tmp/docker-mailserver/postfix-accounts.cf`. The best way to manage accounts is to use the reliable `setup` command inside the container. Just run `docker exec <CONTAINER NAME> setup help` and have a look at the section about subcommands, specifically the `email` subcommand.
```

File: docs/content/config/user-management.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### Configuring RegExp Aliases
+
+Additional regexp aliases can be configured by placing them into `docker-data/dms/config/postfix-regexp.cf`. The regexp aliases get evaluated after the virtual aliases (container path: `/tmp/docker-mailserver/postfix-virtual.cf`). For example, the following `docker-data/dms/config/postfix-regexp.cf` causes all email sent to "test" users to be delivered to `qa@example.com` instead:
+
+```cf
+/^test[0-9][0-9]*@example.com/ qa@example.com
+```
+
+### Address Tags (Extension Delimiters) As an Alternative to Aliases
```
Comment (polarathene) [inline docs/content/config/user-management.md:73]:
```suggestion
### Address Tags (Extension Delimiters) as an alternative to Aliases
```

File: docs/content/config/user-management.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# User Management
+
+## Accounts
+
+Users (email accounts) are managed in `/tmp/docker-mailserver/postfix-accounts.cf`. The best way to manage accounts is to use the reliable `setup` script inside the container. Just run `docker exec <CONTAINER NAME> setup help` and have a look at the section about subcommands, specifically the `email`-subcommand.
+
+### Adding a New Account
+
+#### Via `setup` Inside the Container
```
Comment (polarathene) [inline docs/content/config/user-management.md:9]:
```suggestion
#### Via `setup` inside the container
```

File: docs/content/config/user-management.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+# User Management
+
+## Accounts
+
+Users (email accounts) are managed in `/tmp/docker-mailserver/postfix-accounts.cf`. The best way to manage accounts is to use the reliable `setup` script inside the container. Just run `docker exec <CONTAINER NAME> setup help` and have a look at the section about subcommands, specifically the `email`-subcommand.
+
+### Adding a New Account
```
Comment (polarathene) [inline docs/content/config/user-management.md:7]:
```suggestion
### Adding a new Account
```

**Review by polarathene — CHANGES_REQUESTED** [review]

**Review by polarathene — APPROVED** [review]

**Pull request merged**